__test__/Engineer.test.js

```diff
@@ -0,0 +1,8 @@
+const Engineer = require("../src/Engineer");
+
+describe("Engineer", () => {
```

Comment:
The Engineer, Intern and Manager classes are not testing much more than an instance being created. Please update these tests to look like the Employee one so that we can see the logic working correctly for each of the classes :-)